app/views/searches/dynamic.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for (:head) { javascript_include_tag "dynamic_search" } %>
```

Comment:
Agreed, let's give it some time for beta testing, ensure we can get the whole thing running, etc. before exposing any interfaces to the public, so far as possible.